src/main/groovy/com/github/jengelman/gradle/plugins/shadow/relocation/Relocator.groovy

```diff
@@ -28,11 +28,11 @@ package com.github.jengelman.gradle.plugins.shadow.relocation
 interface Relocator {
     String ROLE = Relocator.class.getName()
 
-    boolean canRelocatePath(RelocatePathContext context)
```

Comment:
this is a breaking change. Is it really necessary? These context objects were introduced a while ago in order to abstract the details away as I was finding that we needed to add additional data for certain features.

Reply:
Oh, I see, you're just removing it from some of the methods.

Reply:
Yes, I removed context only from the methods that aren't using it. It's done to decrease the number of allocations we make and to save a bit of time on creating the builder